code/game/antagonist/outsider/ert.dm

```diff
@@ -39,10 +41,6 @@ var/datum/antagonist/ert/ert
 	player.equip_to_slot_or_del(new /obj/item/clothing/shoes/swat(src), slot_shoes)
 	player.equip_to_slot_or_del(new /obj/item/clothing/gloves/swat(src), slot_gloves)
 	player.equip_to_slot_or_del(new /obj/item/clothing/glasses/sunglasses(src), slot_glasses)
-	player.implant_loyalty(player) //aparantly they are loyalty implanted??
```

Comment:
why would ERT not have loyalty implants? They're given military grade weapons and exosuits, plus an all access pass to a high security research station